spmd/compiler/graph_utils.py

```diff
@@ -24,7 +20,7 @@ class OP(str, Enum):
     PLACEHOLDER = "placeholder"
 
 
-def get_node_tensor_size(node: fx.Node, debug=False) -> int:
+def get_node_tensor_size(node: fx.Node, debug: bool = False) -> Optional[int]:
```

Comment:
This debug is not necessary if it is just used to decide whether to print out the message. Instead, `rank0_debug` should be used instead of `rank0_info`.

Reply:
Also, we should change the function name to `get_node_tensor_numel` to match the PyTorch convention.

Reply:
Great points, thanks!